### ISSUE TITLE
BUG - 3.2.0 missing feature offered in 3.1.1

### ISSUE THREAD

**Issue by eyebank** [standup-raven/standup-raven#230]
**Summary**
I appreciate your work:

Problem:
1) in 3.2.0 the command: /standup report public does not work. a hint states must be in proper format DD-MM-YYYY
2) in 3.1.1 the command: /standup report public will display your report immediately without adding date.
Please consider bringing back 3.1.1 standup report public functionality. 
3) the raven icon is missing from header in channel.

I have rolled back to 3.1.1 from 3.2.0

**Steps To Reproduce**

**Expected behavior**
What you expected to happen.
1) in 3.1.1 the command: /standup report public will display your report immediately without adding date.
2) I expected the raven icon to be visible in the channel header.

**Observed Behavior**
What actually happened.
in 3.2.0 the command: /standup report public does not work. a error/hint states must be in proper format DD-MM-YYYY no report is published
and System
13:50(Only visible to you)
Please specify report format and dates to generate report for.

2) No raven icon

**Screenshots**
![3_1_1](https://user-images.githubusercontent.com/8021595/101076447-63fc7c80-3571-11eb-8f0f-ac8e7fa918c9.jpg)
![3_2_0](https://user-images.githubusercontent.com/8021595/101076459-67900380-3571-11eb-9b0a-d299449e9123.jpg)

Hope this helps!!!

**Comment by harshilsharma63** [comment]
@eyebank yeah, I guess the old defaults were simpler. Will restore those in the next release.
